lib/miq_expression/tag.rb

```diff
@@ -19,6 +19,7 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
+    @managed = managed
     @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
```

Comment:
I think `namespace` is not quite right - it includes more than the namespace. Could this be changed so that you're encapsulating this knowledge (which namespace to use)? `namespace` is its public API though, so I don't know how far reaching this would be